app/src/main/java/com/futurice/freesound/feature/search/SearchFragmentViewModel.java

```diff
@@ -56,6 +57,7 @@
         return searchDataModel.getSearchStateOnceAndStream()
                               .map(SearchState::results)
                               .map(it -> it.map(SearchFragmentViewModel::wrapInDisplayableItem))
+                              .delay(1, TimeUnit.SECONDS)
```

Comment:
This should trigger the lint warning because I haven't used `delay` with a Scheduler.